src/Tools/dotnet-gcdump/DotNetHeapDump/GCHeapDump.cs

```diff
@@ -39,6 +39,15 @@ public static void WriteMemoryGraph(MemoryGraph graph, string outputFileName, st
         dumper.Write(outputFileName);
     }
 
+    /// <summary>
+    /// Reads the memory graph from a .gcdump file
+    /// </summary>
+    public static GCHeapDump ReadMemoryGraph(string inputFileName)
```

Comment:
nit: I'd like to leave the code in here as unmodified as possible to make keeping it in sync with Microsoft/PerfView easier.  Could we make this an extension method elsewhere instead?